widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/parse/Deserializers.kt

```diff
@@ -47,7 +46,8 @@ internal class ColorDeserializer : JsonDeserializer<ColorRemoteConfig?> {
     override fun deserialize(
         json: JsonElement, typeOfT: Type, context: JsonDeserializationContext
     ): ColorRemoteConfig? = tryOrNull {
-        ColorRemoteConfig(SystemColor.parseColor(json.asString))
+        val rawColor = json.asString.run { if (startsWith("#")) this else "#$this" }
+        ColorRemoteConfig(SystemColor.parseColor(rawColor))
```

Comment:
I rethought it, this is not expected behavior and I added this for cases then # was accidentally omitted, but I think this behaviour can rather confuse integrators. It would be better to remove it back.